src/mlbstandings/web.py

```
@@ -1,17 +1,27 @@
+ 
 class Web:
     @staticmethod
+    def retryable(code: int) -> bool:
+        return False  # TODO
+    @staticmethod
     @backoff.on_exception(
         backoff.expo,
-        (requests.exceptions.HTTPError, requests.exceptions.ConnectionError, requests.exceptions.Timeout),
+        (requests.exceptions.HTTPError, urllib.error.HTTPError, requests.exceptions.ConnectionError, requests.exceptions.Timeout),
         max_time=600,
         giveup=lambda e : isinstance(e, requests.exceptions.HTTPError) and e.response.status_code not in set([429, 500, 503]),
```

Ruff (0.11.9)

Unnecessary list literal (rewrite as a set literal)

Rewrite as a set literal

(C405)